src/indexset.jl

```diff
@@ -63,108 +60,65 @@ IndexSet(f::Function, ::Order{N}) where {N} =
   IndexSet(ntuple(f, Val(N)))
 
 # Definition to help with generic code
-const Indices{N, IndexT} = Union{IndexSet{N,
-                                          IndexT,
-                                          NTuple{N, IndexT}},
-                                 NTuple{N, IndexT}}
-
-function (IndexSet{N, IndexT}(inds)::IndexSet{N, IndexT, NTuple{N, IndexT}}) where {N, IndexT}
-  data = NTuple{N, IndexT}(inds)
-  return IndexSet{N, IndexT, NTuple{N, IndexT}}(data)
-end
+const Indices{N, IndexT} = Union{IndexSet{IndexT},
```

Comment:
This shouldn't need the type parameter `N` anymore. I forget if this type is even being used anywhere...

Reply:
It is used in `src/itensor.jl`.